edge-modules/MetricsCollector/docker/windows/arm32v7/Dockerfile

```diff
@@ -0,0 +1,11 @@
+ARG base_tag=1.0.3-windows-arm32v7
+ARG base_registry
+FROM ${base_registry}/azureiotedge/azureiotedge-module-base:${base_tag}
+
+ARG EXE_DIR=.
```

Comment:
why no MODULE_NAME here?

Reply:
I'm not sure what MODULE_NAME means in this context. The dockerfile here follows the same format as in load-gen.

Reply:
Please help to follow up with Varun to remove these from linux docker files or add it to Windows Docker file.